client/gefyra/cluster/manager.py

```diff
@@ -25,6 +25,37 @@
 logger = logging.getLogger(__name__)
 
 
+def handle_create_namespace(config: ClientConfiguration, retries=10, wait=3):
+    counter = 0
+    while counter < retries:
```

Comment:
There is no case handling the timeout here (`counter >= retries`). It's likely Gefyra will crash somewhere else.

Reply:
Added a custom exception when many 409 errors occurs.